sway-lsp/tests/lib.rs

```diff
@@ -165,6 +165,46 @@ fn did_change() {
     });
 }
 
+#[test]
+fn sync_with_updates_to_manifest_in_workspace() {
```

Comment:
nit: propose using `tempfile` dep to create tempdir - to ensure proper cleanup in case user exits or there is error mid-test
Not a biggy though

Reply:
nice, yeah i'm generally in favour. In this instance I already have a local copy for interactive testing so probably makes sense the way it is.